src/components/tests/GeoPushpinChart.spec.tsx

```diff
@@ -0,0 +1,208 @@
+// (C) 2019-2020 GoodData Corporation
+import * as React from "react";
+import { shallow } from "enzyme";
+import { factory } from "@gooddata/gooddata-js";
+import { AFM, VisualizationObject } from "@gooddata/typings";
+
+import { GeoPushpinChart } from "../GeoPushpinChart";
+import { IGeoConfig } from "../../interfaces/GeoChart";
+
+describe("Geo Pushpin Chart", () => {
+    const size: VisualizationObject.IMeasure = {
```

Comment:
You can use this for short
```
import { Model } from "../..";
const m1 = Model.measure("measure_size");
const a1 = Model.attribute("attribute_segmentBy");
```

Reply:
thanks. It is another way to create mock data unit-test. I will take note this.